inbc-program/inbc/command/command.py

```diff
@@ -129,7 +129,7 @@ def invoke_update(self, args: Any) -> None:
 
         @param args: arguments passed to command-line tool.
         """
-        super()._send_manifest(args, RESTART_CHANNEL)
+        super()._send_manifest(args, RESTART_CMD_CHANNEL)
 
     def search_response(self, payload: Any) -> None:
         """Search for keywords in response message
```

Comment:
**<h2>$\color{#FFD700}Coding Style$</h2>**
**Suggestion:** The print statements in the search_event method of the Command class and its subclasses are used for logging purposes. Consider using the logger for consistency with the rest of the codebase and to provide more control over the logging output.
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
if search_keyword(payload, ["/usr/bin/mender -install"]):
    logger.info("Flashing mender file. This will take several minutes...")
```